pkg/serverless/logs/logs.go

```diff
@@ -177,18 +172,13 @@ func (l *logMessage) UnmarshalJSON(data []byte) error {
 				} else {
 					log.Error("LogMessage.UnmarshalJSON: can't read the metrics object")
 				}
-			case logTypePlatformRuntimeDone:
-				if status, ok := objectRecord["status"].(string); ok {
-					l.objectRecord.runtimeDoneItem.status = status
-				} else {
-					log.Debug("Can't read the status from runtimeDone log message")
-				}
 			}
 		} else {
 			log.Error("LogMessage.UnmarshalJSON: can't read the record object")
 		}
 	default:
 		// we're not parsing this kind of message yet
+		// platform.extension, platform.logsSubscription, platform.fault
```

Comment:
This way, if someone is searching the code base for one of these log message types looking for information on what the extension does with them, they will get their answer much faster.